runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectOptions.java

```diff
@@ -49,10 +49,18 @@
 
   @Default.Boolean(true)
   @Description(
-      "Controls whether the runner should ensure that all of the elements of every "
+      "Controls whether the DirectRunner should ensure that all of the elements of every "
           + "PCollection are not mutated. PTransforms are not permitted to mutate input elements "
           + "at any point, or output elements after they are output.")
   boolean isTestImmutability();
 
   void setTestImmutability(boolean test);
+
+  @Default.Boolean(true)
+  @Description(
+      "Controls whether the DirectRunner should ensure that all of the elements of every "
+          + "PCollection are encodable. All elements in a PCollection must be encodable.")
+  // All non-null elements will be encoded if isTestImmutability set to true
```

Comment:
I'm confused by this extra comment is it saying:

1. If isTestEncodability is true, not all elements will be encoded, unless isTestImmutability is also true?
2. Even if isTestEncodability is false, all non-null elements will still be encoded if isTestImmutability is true?